packages/devel/swig/package.mk

```diff
@@ -3,13 +3,14 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="swig"
-PKG_VERSION="4.0.2"
-PKG_SHA256="d53be9730d8d58a16bf0cbd1f8ac0c0c3e1090573168bfa151b01eb47fa906fc"
+PKG_VERSION="4.1.0"
+PKG_SHA256="d6a9a8094e78f7cfb6f80a73cc271e1fe388c8638ed22668622c2c646df5bb3d"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.swig.org"
 PKG_URL="${SOURCEFORGE_SRC}/swig/${PKG_NAME}-${PKG_VERSION}.tar.gz"
```

Comment:
There is https://github.com/swig/swig too, also looks like it supports cmake these days.

Reply:
I left as SF - just because that is where https://www.swig.org/download.html is still pointing. Hopefully moves to GH by next release. Left it running configure, as the CMake is not quite equal with configure, and leave it to soak - some CMake fixes already in master